### PR TITLE
Add 5 icons, update 1 icon

### PR DESCRIPTION
**Pull request by dimassasin1**
diia - I mirrored the letter "R", because there should be a cyrillic "Я".
[easyway](https://play.google.com/store/apps/details?id=com.eway)
[helsi](https://play.google.com/store/apps/details?id=helsi.me)
[lifecell](https://play.google.com/store/apps/details?id=com.life.my)
[monobank](https://play.google.com/store/apps/details?id=com.ftband.mono)
[pcu](https://play.google.com/store/apps/details?id=com.d2.pcu)

### CODE REVIEW

**Comment by Donnnno** [comment]
> diia - I mirrored the letter "R", because there should be a cyrillic "Я".

Sharp!

The icons look good, but you forgot to update the appfilter.xml with all the entries of your icons!

**Comment by Donnnno** [comment]
I've added easyway, helsi and monobank to the appfilter, the other icons (lifecell & pcu) weren't in the request. 

You can gather the info you need with the fantastic Icon Request app: https://github.com/Kaiserdragon2/IconRequest/releases

**Comment by dimassasin1** [comment]
`<appfilter>

 
 <item component="ComponentInfo{com.d2.pcu/com.d2.pcu.MainActivity}" drawable="_"/>

 
 <item component="ComponentInfo{com.life.my/com.efectura.lifecell2.ui.activity.SplashActivity}" drawable="my_lifecell"/>

</appfilter>`

File: other/appfilter.xml [hunk 1]
```diff
@@ -26,7 +26,16 @@
...
+      
+  <!-- monobank -->
+  <item component="ComponentInfo{com.ftband.mono/com.ftband.app.StartActivity.Mono}" drawable="monobank" />
+      
+  <!-- HELSI -->
+  <item component="ComponentInfo{helsi.me/com.zoontek.rnbootsplash.RNBootSplashActivity}" drawable="helsi" />
+      
+  <!-- EasyWay -->
+  <item component="ComponentInfo{com.eway/com.eway.android.activity.MainActivity}" drawable="letter_uppercase_square_e" />
+
```
Comment (dimassasin1) [inline other/appfilter.xml:38]: <item component="ComponentInfo{com.d2.pcu/com.d2.pcu.MainActivity}" drawable="_"/>

 
 <item component="ComponentInfo{com.life.my/com.efectura.lifecell2.ui.activity.SplashActivity}" drawable="my_lifecell"/>

Comment (dimassasin1) [inline other/appfilter.xml:38]: <item component="ComponentInfo{com.d2.pcu/com.d2.pcu.MainActivity}" drawable="_"/>

 
 <item component="ComponentInfo{com.life.my/com.efectura.lifecell2.ui.activity.SplashActivity}" drawable="my_lifecell"/>

Comment (dimassasin1) [inline other/appfilter.xml:38]:
```

 <!-- Моя Церква -->
 <item component="ComponentInfo{com.d2.pcu/com.d2.pcu.MainActivity}" drawable="_"/>

 <!-- My lifecell -->
 <item component="ComponentInfo{com.life.my/com.efectura.lifecell2.ui.activity.SplashActivity}" drawable="my_lifecell"/>

```

**Comment by Donnnno** [comment]
getting some merge errors, so added everything in another PR, thanks for the icons!

**Pull request closed without merging**